tesseract/tesseract_planning/tesseract_command_language/include/tesseract_command_language/composite_instruction.h

```diff
@@ -106,7 +105,16 @@ class CompositeInstruction
   using reverse_iterator = typename std::vector<value_type>::reverse_iterator;
   /** const_reverse_iterator */
   using const_reverse_iterator = typename std::vector<value_type>::const_reverse_iterator;
+#endif
+
+#ifndef SWIG
```

Comment:
This could be the way it is showing the changes but this does not appear to have an #endif before the next #ifndef.

Reply:
You are right, I will fix this.

Reply:
Fixed